### DISCUSSION TITLE
Fan speed settings lost after a reboot

### DISCUSSION THREAD

**Discussion by 62Maurizio** [Rem0o/FanControl.Releases#2819] [General]
Hi 
Env  
SW Win10 
Hw  cpu 6 cores  with a fan tachiometer controlled  , gpu with a fan  tachiometer controlled + 2 chassis fan without tachiometer 
Machine is perfectly cleaned, i installed this pgm because cpu fan run very fast getting noise  
Pgm installs without problem and it test gpu and cpu fan first increasing speed then decreasing speed, when cpu speed go at 80% instead of initial 89% noise stop... after the setup cpufan speed isset a very low rpm and  alla is ok...
then i change configuration deselectingsearch for update, and pgm prompt me to save configurastion..
fan speed remains very low, i checked thermal  temperature and is very  low...
but if i reboot pc it seems program is no more working, gpu speed is at 20% and it get not noise , cpu speed returns to 89% fan turns at 3400 rpm getting noise... 
there is no cpu activity so there is no need that fan turn so fast ...
why fancontrol does not  work anymore after a reboot ?

**Comment by Rem0o** [comment]
Show your configuration with a screenshot.

**Comment by 62Maurizio** [comment]
Hi
i upload 5 screenshot they are named with a numeric prefix 
1) after setup fancontrol perform calibration and show this window
2) screenshot 2 is taken 5 minutes after setup, as you can see fancontrol decrease fan2 speed 
3) then i saveconfiguration and i reboot, after rebooting fan control starts, and after few minutes i take screenshot 3  
4) a this point i enable softwarespeed control and i save configuration. but fan speed does not decrease
5) i reboot again but situation does not changes 
my motherboars is asrock 990fx extreme 3, and the bios is already at the last level (1.90)
as you can see fan2 is the cpu fan andit turns at 3400 rpm, is enough to decrese this speed to 3100 rpmand the noise is relieved
thanks in advance for your support 
Maurizio
  
![1AfterInstallatioSetup](https://github.com/user-attachments/assets/8476490d-7631-4823-b930-8ee87425feff)
![2after5minutes](https://github.com/user-attachments/assets/7ee9294a-c9d6-45be-af66-abf85c02a7e8)
![3AfterReboot](https://github.com/user-attachments/assets/118e61c3-1b57-4dd1-9821-a0e5d46dbeaa)
![4EnableSwControl](https://github.com/user-attachments/assets/6955ff48-362a-48ef-ae22-7005b60f01bb)
![5afterSecondReboot](https://github.com/user-attachments/assets/a8545d20-3efa-45eb-bb6f-af8d3fd6acdd)

**Comment by Rem0o** [comment]
You have no curve assigned to your controls.  Effectively the motherboard still has control.  You need to use the "+" button at the bottom right and assign that fan curve to each control (the "Curve" dropdown).

**Comment by 62Maurizio** [comment]
Hi... to do not waste time ...
the plus "+"button opens a menu... with the followin entries 
Auto - Sync - Flat - Trigger - Mix - Graph - Linear - Mix - Time Average - File - Offset 
What to choose to set up ?
Thanks in advance
